static/js/containers/UserMenu.js

```diff
@@ -28,8 +28,12 @@ class LoginButton extends React.Component {
       <LinkContainer to={{ pathname: '/dashboard' }} active={false}>
         <SplitButton
           title={title}
-          bsStyle="danger"
-          id="logout-button">
+          bsStyle="danger">
+          <MenuItem
+            href={`/users/${profile.username}`}
+            eventKey="users_page">
```

Comment:
i don't really know what this `eventKey` property is intended for. the logout MenuItem has it but I didn't see what its purpose was